src/core/continuous_space.jl

```diff
@@ -146,3 +149,35 @@ function move_agent!(agent::A, model::ABM{A, S, F, P}) where {A<:AbstractAgent,
   end
   return agent.pos
 end
+
+function kill_agent!(agent::AbstractAgent, model::ABM{A, S}) where {A, S<:ContinuousSpace}
+  DBInterface.execute(model.space.deleteq, (agent.id,))
+  delete!(model.agents, agent.id)
+  return model
+end
+
+function genocide!(agentIDs::AbstractArray{Int}, model::ABM{A, S}) where {A, S<:ContinuousSpace}

```

Comment:
Maybe it makes sense to use the already existing and intuitive API of `genocide!`:  https://juliadynamics.github.io/Agents.jl/stable/api/#Agents.genocide!  and have the input arguments be the exact same 3 combinations?